File: acm_statistic_backend/api/api/DTO/performance/compagnie/RepartitionTypeCompagnieMoisDTO.cs
namespace api.DTO.performance.compagnie
{
    public class RepartitionTypeCompagnieMoisDTO
    {
        public int Mois {  get; set; }
        public string Mois_Lettre { get; set; }
        public string Type_trafic { get; set; }
        public decimal? Montant { get; set; }
        public decimal? Part {  get; set; }
        public string Compagnie { get; set; }
        public int Vols { get; set; }
        public decimal? Pourcentage_Variation { get; set; }
    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/performanceCompagnie/RepartitionTypeCompagnieMoisDTO.ts
export class RepartitionTypeCompagnieMoisDTO{
    mois: number;
    mois_lettre: string;
    montant?: number; 
    compagnie: string;
    vols: number;
    part?: number;
    pourcentage_variation?: number;

    constructor(init?: Partial<RepartitionTypeCompagnieMoisDTO>) {
    Object.assign(this, init);
    }   
}